### DISCUSSION TITLE
Pull Shark details not correct

### DISCUSSION THREAD

**Discussion by r1cs** [community/community#18213] [Other Feature Feedback, Questions, & Ideas] [answered]
The beta version of `Pull Shark` shows the wrong merged pr num for me

**Comment by DavidYang2149** [comment, marked as answer]
Hi~!
Pull Shark seem to be check your PR record and show Level (not your PR count status)
x1 - over 2 PR
x2 - over 16 PR
x3 - over 128 PR
x4 - over 1024 PR

so, your level x2.
In that case, your PR count is 16~128.

**Reply by aleebaba3594** [reply]
mark my reply i want get badge

**Comment by bagder** [comment]
It also probably assumes that all the PRs were merged on GitHub with the **merge** button, right? I've had way more than 1024 PRs merged, but we don't use the merge button in my primary projects so they're not counted by the badge system.
